CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+## [0.8.2] - 2020-06-23
 ### Added
 - Added ability to search for secrets.yml going up, starting from the current working directory
```

Comment:
The changelog from this version appears to be missing some entries - can you review the [commits since the last release]() and ensure that everything user-impacting is there?

I think it's mainly the changes from https://github.com/cyberark/summon/pull/138 that are missing

Reply:
I believe you're correct, I can't see any other user facing changes. Thanks for pointing this out!